services/graph/pkg/service/v0/users.go

```diff
@@ -320,16 +337,18 @@ func (g Graph) GetUser(w http.ResponseWriter, r *http.Request) {
 				logger.Debug().Err(err).Interface("id", sp.Id).Msg("error calling get quota on drive")
 			}
 			d.Quota = &quota
-			if slices.Contains(sel, "drive") || slices.Contains(exp, "drive") {
-				if *d.DriveType == "personal" {
+			if d.GetDriveType() == "personal" && sp.GetOwner().GetId().GetOpaqueId() == user.GetId() {
+				if listDrive {
 					user.Drive = d
 				}
 			} else {
-				drives = append(drives, *d)
-				user.Drives = drives
+				if listDrives {
+					user.Drives = append(user.Drives, *d)
+				}
```

Comment:
Just for clarification: This means that personal spaces do not appear in the `drives` list. I can only get them as single `drive`. Is that the intended behaviour?

Reply:
Yes, there is always only one personal drive you are the owner of: /me/drive

As a secretary you might get access to other personal drives but they will only show up in /me/drives.

The admin might create anotherdrive that is owned by you. I would still use type 'project' for that. We should reserve type 'personal' only for the users personal drive.